spec/support/utils/profile-testing-util.js

```diff
@@ -0,0 +1,30 @@
+const enums = require("../../../lib/main/constants/enumConstants.js");
+const { generateRandomString,} = require("./general-testing-util.js");
+const generateRandomProfile = (username, createdAt, age, bmi, height, weight, shouldParse = true) => {
+const generateRandomDate = () => {
```

Comment:
What is this used for?

Reply:
we also use a specific format for date look at the moment code. If this is a general function it should go in some sort of general util class